### PR TITLE
Update proj1rubric.md

### PR DESCRIPTION
**Pull request by mithildave**
Added many new rubrics.

### CODE REVIEW

File: docs/proj1rubric.md [hunk 1]
```diff
@@ -1,45 +1,87 @@
...
+3| Number of commits: by different people|2|https://github.com/shahrk/feature-hunt/graphs/commit-activity|
+4| Issues reports: there are **many**|3|https://github.com/shahrk/feature-hunt/issues|
+5|  issues are being  closed|3|evidence in GH|
+6| DOI badge: exists |3|in GH Readme|
+7|Docs: doco generated , format not ugly|3 |We prepared Readme file which is easily readbale and looks good.|
+8|Docs: what: point descriptions of each class/function (in isolation) |3|Refer to this example commit: https://github.com/shahrk/feature-hunt/commit/81690d9ee9c4a0d577e301ee3c0c0cadc9d1473e|
+9|Docs: how: for common use cases X,Y,Z mini-tutorials showing worked examples on how to do X,Y,Z|3|Added Screenshots/Video explainign the worked example|
+10|Docs: why: docs tell a story, motivate the whole thing, deliver a punchline that makes you want to rush out and use the thing|3| Logo, Description amd story telling is done in Readme|
+11|Docs: short video, animated, hosted on your repo. That convinces people why they want to work on your code.|3| in GH Readme|
+12| Use of version control tools|0| Small scale project, can use in future |
```
Comment (shahrk) [inline docs/proj1rubric.md:20]: Git is a version control tool - 0 doesn't make sense

File: docs/proj1rubric.md [hunk 2]
```diff
@@ -1,45 +1,87 @@
...
+13| Use of  style checkers |3| in GH|
+14| Use of code  formatters. |3| in GH|
+15| Use of syntax checkers. |3|in GH|
+16| Use of code coverage |0||
```
Comment (shahrk) [inline docs/proj1rubric.md:24]: We have code coverage. This should be 3

File: docs/proj1rubric.md [hunk 3]
```diff
@@ -1,45 +1,87 @@
...
+31|case studies to show how your software has been used by yourself and others|3|Screenshots in https://github.com/shahrk/feature-hunt/blob/main/README.md|
+32|Documentation includes a "quick start" guide that provides a short overview of how to use your software with some basic examples of use.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+33|Documentation provides clear, step-by-step instructions on how to deploy and use your software.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+34|A comprehensive guide to all your software’s commands, functions and options.|3|https://github.com/shahrk/feature-hunt/blob/main/README.md|
+35|If your software can be used as a library, package or service by other software, do you provide comprehensive API documentation?|3|Yes, Full documentation is present in GH readme|
+36|Troubleshooting information that describes the symptoms and solutions for problems.|1|Kept as future scope|
+37|Documentation stored under revision control with your source code.|1|Its considered as Future Scope|
```
Comment (shahrk) [inline docs/proj1rubric.md:45]: Documentation is on GitHub so it is by default stored under revision control

**Review by shahrk — CHANGES_REQUESTED** [review]
Please address the comments

File: docs/proj1rubric.md (hunk 3 above)
Comment (mithildave) [reply docs/proj1rubric.md]: Modified in the new commit

**Pull request merged**